packages/core/components/DraggableComponent/index.tsx

```diff
@@ -177,8 +177,8 @@ export const DraggableComponent = ({
             </div>
           )}
 
-          <div className={getClassName("overlay")} />
-          <div className={getClassName("contents")}>{children}</div>
+          <div className={getClassName("overlay")} style={{ ...(isInteractive && { cursor: 'default', background: 'transparent' }) }} />
+          <div className={getClassName("contents")} style={{ ...(isInteractive && { pointerEvents: 'auto' }) }}>{children}</div>
```

Comment:
These would probably be better as a CSS class, passed to the `getClassName` function [here](https://github.com/measuredco/puck/pull/674/files#diff-d625e3b274e5aa193679313129123d868084442be8eaa76119437f5ceb0f6ebaR124).

Reply:
Done

Reply:
done